CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Lao numeral system
+### Changed
+- Test system modified
```

Comment:
Is this for a differnet PR that's added here?
Also, idk why Codacy's Semgrep test is failing. It's referring to `meta.yml` first line that hasn't been changed in this PR. So either it's a false positive or a result of a prior PR that I wasn't aware of.